lib/core/bytes.nit

```diff
@@ -538,6 +538,12 @@ class Bytes
 		length += cln
 	end
 
+	redef fun has(c)
+	do
+		if not c isa Int then return false
+		return super(c&255)
```

Comment:
Given the semantics, maybe we should have a comment for this specialisation stating we do truncate `c` to 8 bits so we can avoid surprises in the future

Reply:
I agree, the behavior make sense but should be documented